bower.js

```diff
@@ -116,9 +116,12 @@ exports.translate = function(load){
 	// Get the AMD dependencies
 	var amdDeps = [];
 	for(var dep in deps) {
-		amdDeps.push(
-			bowerPath + "/" + dep + "/bower.json!bower"
-		);
+		// add a check of bower.system.bowerIgnore before pushing dep
+		if (!bower.system || !bower.system.bowerIgnore || bower.system.bowerIgnore.indexOf(dep) === -1) {
```

Comment:
I would assign a `bowerIgnore` above the for loop to avoid checking for bower.system and bower.system.bowerIgnore every time.

```
var bowerIgnore = (bower.system && bower.system.bowerIgnore) || [];
```